src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.ilproj

```diff
@@ -95,7 +95,7 @@ $(ExtraMacros)
   </Target>
 
   <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
-    <ProjectReference Include="..\ver\System.Runtime.CompilerServices.Unsafe.Versioning.csproj"
+    <ProjectReference Include="..\ver\Versioning.csproj"
```

Comment:
Please set `AssemblyName` in `Versioning.csproj` so that the OriginalFileName field in version resources is correct.

Reply:
Resolving - missed the addition in the props file.  We can follow up on Viktor's comment.